### ISSUE TITLE
Feature Request: Motion Detection on Camera

### ISSUE THREAD

**Issue by chrisfromthelc** [Automattic/biab-plugin#38]
This could be useful for situations where the user may want to use is as a pseudo-security camera/action camera, taking shots on motion detection instead of simple scheduled shots.

**Comment by johngodley** [comment]
Yep it'd be a great feature. Using [Motion](http://www.lavrsen.dk/foswiki/bin/view/Motion/MotionGuideInstallation) I think this would be very doable.
